cmd/juju/interact/pollster_test.go

```diff
@@ -473,7 +473,7 @@ func (PollsterSuite) TestQueryObjectSchema(c *gc.C) {
 	schema := &jsonschema.Schema{
 		Type: []jsonschema.Type{jsonschema.ObjectType},
 		Properties: map[string]*jsonschema.Schema{
-			"numbers": &jsonschema.Schema{
```

Comment:
I didn't realize it would automatically create a pointer to a struct. Interesting.

Reply:
Neither did I, until the spec said...

Reply:
No me... curious.